src/pombot/pom.js

```diff
@@ -1,8 +1,10 @@
-import states from './states';
+import {states} from './pomConfig';
+import moment from 'moment';
 
 export default class Pom {
   constructor(options = {}) {
     this.state = states.NOT_RUNNING;
+    this.is_im = options.is_im;
```

Comment:
it *was* how i checked what kind of pom (dm vs channel) it was so i can have it say the proper command to start, but then last night i learned about the arguments i could pass into createCommand so i just removed it!